src/config.rs

```diff
@@ -153,4 +152,62 @@ impl Config {
 
         Ok(config)
     }
+
+    pub fn from_args(_args: std::env::Args) -> Result<Config, Error> {
```

Comment:
What's the point of this if "_args" is never used?

Reply:
Good question.  I think that was a holdover from when I first wrote this.  The clap examples did something like that.  Either way I removed it.